install/aws/aws-chatbot/install.yml

```diff
@@ -0,0 +1,13 @@
+id: aws-chatbot
```

Comment:
Install plans are currently deprecated in favor of data sources, no need to add this:
https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-chatbot-monitoring-integration/